tests/unit/test_betabeatsrc_output_converter.py

```diff
@@ -0,0 +1,206 @@
+from pathlib import Path
+
+import pytest
+import tfs
+
+from omc3.definitions.constants import PLANES
+from omc3.optics_measurements.constants import (
+    AMPLITUDE,
+    AMP_BETA_NAME,
+    BETA_NAME,
+    DELTA,
+    DISPERSION_NAME,
+    ERR,
+    IMAG,
+    MDL,
+    NORM_DISP_NAME,
+    ORBIT_NAME,
+    PHASE,
+    PHASE_NAME,
+    REAL,
+    TOTAL_PHASE_NAME,
+)
+from omc3.scripts.betabeatsrc_output_converter import converter_entrypoint
+from tests.conftest import cli_args
+
+INPUT_DIR = Path(__file__).parent.parent / "inputs"
+BBRSC_OUTPUTS = INPUT_DIR / "bbsrc_output_converter"
+
+
+# ----- Tests ----- #
+
+
+@pytest.mark.basic
+@pytest.mark.parametrize("suffix", ["", "_free", "_free2"])
+def test_betabeatsrc_output_converter(tmp_path, suffix):
+    converter_entrypoint(inputdir=str(BBRSC_OUTPUTS), outputdir=str(tmp_path), suffix=suffix)
+    _assert_correct_files_are_present(tmp_path)
+
+    for plane in PLANES:
+        _assert_correct_beta_amp_columns(tmp_path, plane)
+        _assert_correct_beta_phase_columns(tmp_path, plane)
+        _assert_correct_phase_columns(tmp_path, plane)
+        _assert_correct_total_phase_columns(tmp_path, plane)
+        _assert_correct_closed_orbit_columns(tmp_path, plane)
+        _assert_correct_dispersion_columns(tmp_path, "X")
+    _assert_correct_normalized_dispersion_columns(tmp_path, "X")  # no norm disp in Y plane
+
+    for rdt in ["1001", "1010"]:
+        _assert_correct_coupling_columns(tmp_path, rdt)
+
+
+@pytest.mark.basic
+@pytest.mark.parametrize("suffix", ["", "_free", "_free2"])
+def test_betabeatsrc_output_converter_commandline(tmp_path, suffix):
+    with cli_args(
+        "--inputdir", str(BBRSC_OUTPUTS.absolute()), "--outputdir", str(tmp_path), "--suffix", suffix
+    ):
```

Comment:
is there a way to put this in a parameterized fct, and then use it on measure optics/modelcreator tests, in conjunction with the regular call as python fct .. looking at the recent issue with an erroneous type in an entrypoint

Reply:
Considering this output of this is not for measure optics, I don't know. Maybe I misunderstood your comment.

Reply:
currently, in a majority of the tests we only provide input via a function call, e.g. omc3.hole_in_one.optics_entrypoint(...)
I was wondering if we could create tests to use cli_args, so checking that it also works via a commandline call ..
but in other to avoid duplication, I was wondering if we can use pytest.parameterize(cmdline, [True, False]) together with cli args on top of some tests to check that commandline input also works
(not to be put in this PR, just a general Q)

Reply:
Indeed, not in this PR but the idea is good. I will have a look over the weekend.